admin/class-yoast-form.php

```diff
@@ -417,11 +480,11 @@ public function hidden( $var, $id = '' ) {
 	 *
 	 * @since 2.0
 	 *
-	 * @param string $field_name     The variable within the option to create the select for.
+	 * @param string $var            The variable within the option to create the select for.
 	 * @param string $label          The label to show for the variable.
 	 * @param array  $select_options The select options to choose from.
 	 */
-	public function select( $field_name, $label, array $select_options ) {
+	public function select( $var, $label, array $select_options ) {
```

Comment:
Please name this `$variable` instead of a shorten version for it.

Reply:
I think `$var` or my earlier suggestion `$variable` it too vague. It represents an value in the options or a key.

Reply:
Given that the names for the same thing for all other methods in the class are `$var`, I think we should be consistent here. I agree that we can find a better name, but that should happen for the entirety of this class IMO, preferably in a separate issue.